airbyte-integrations/connectors/destination-bigquery-denormalized/Dockerfile

```diff
@@ -4,7 +4,9 @@ WORKDIR /airbyte
 
 ENV APPLICATION destination-bigquery-denormalized
 
-ADD build/distributions/${APPLICATION}*.tar /airbyte
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
 
-LABEL io.airbyte.version=0.1.11
+RUN tar xf ${APPLICATION}.tar --strip-components=1
```

Comment:
Should also remove the `RUN` command.

```suggestion
```

Reply:
This change ruins integration tests.
https://github.com/airbytehq/airbyte/runs/4570205306?check_suite_focus=true
I removed it back.